### PR TITLE
Rename isPOD to isPod to silence deprecation warnings

### PR DESCRIPTION
**Pull request by erik-krogh**
## Description

This PR targets the `next` branch.

I want to rename a bunch of acronyms to use camelCase/PascalCase to follow our own style guide for QL.  
(See this draft PR: https://github.com/github/codeql/pull/10153).   

When I do that 4 tests in this repo start to fail because of deprecation warnings because if the `idPOD()` predicate. 
I'll let them fail for now, and after I've merged the semmle-code PR I'll fix the deprecation warnings. 

## Change request type
 - [x] Query files (`.ql`, `.qll`, `.qls` or unit tests)

## Rules with added or modified queries

 - [x] No rules added

## Release change checklist

A change note ([development_handbook.md#change-notes](https://github.com/github/codeql-coding-standards/blob/main/docs/development_handbook.md#change-notes)) is required for any pull request which modifies:

 - The structure or layout of the release artifacts.
 - The evaluation performance (memory, execution time) of an existing query.
 - The results of an existing query in any circumstance.

If you are only adding new rule queries, a change note is not required.

_**Author:**_ Is a change note required? 
  - [ ] Yes
  - [x] No

_**Reviewer:**_ Confirm that either a change note is not required or the change note is required and has been added.
  - [ ] Confirmed

### CODE REVIEW

**Comment by lcartey** [comment]
I think we would be better changing our use of `isPOD` to `isPod` on the `next` branch, rather than trying put workarounds in here. @jketema what do you think?

**Comment by jketema** [comment]
The policy is that the internal coding standards test may temporarily be broken while we're in the middle for fixing things. So the breakage is no problem. It should not be blocking merging, if it does we have configured something wrongly.

So I agree with @lcartey that we do not need a workaround. That does mean that the CI here will temporarily break until we update to the next release on the `next` branch, but that should also not considered to be blocking.

**Comment by jketema** [comment]
TL;DR: Just rename `isPOD` to `isPod` and let us know when the internal CI checks should be expected to be green again.

**Comment by jketema** [comment]
Also, yes, this should target `next` and not `main` as `next` is the branch we use in internal testing.

**Comment by erik-krogh** [comment]
I've merged the [github/codeql](https://github.com/github/codeql/pull/10153) and [semmle-code](https://github.com/github/semmle-code/pull/43527) PRs that do the rename from `isPOD()` to `isPod()`.   
Now I think we just wait for the next release that includes that rename.

**Review by jketema — APPROVED** [review]

**Comment by jketema** [comment]
Merging this now to get the internal CI up and running again. We'll fix the one here when 2.11.0 gets released.
